packages/web-api/src/methods.ts

```diff
@@ -1456,6 +1475,28 @@ export interface AppsEventAuthorizationsListArguments
   event_context: string;
 }
 cursorPaginationEnabledMethods.add('apps.event.authorizations.list');
+
+export interface AppsManifestCreateArguments extends WebAPICallOptions, TokenOverridable {
+  manifest: string;
+}
+
+export interface AppsManifestDeleteArguments extends WebAPICallOptions, TokenOverridable {
+  app_id: string;
+}
+
+export interface AppsManifestExportArguments extends WebAPICallOptions, TokenOverridable {
+  app_id: string;
+}
+
+export interface AppsManifestUpdateArguments extends WebAPICallOptions, TokenOverridable {
+  app_id: string;
+  manifest: string;
+}
+
+export interface AppsManifestValidateArguments extends WebAPICallOptions, TokenOverridable {
+  manifest: string;
```

Comment:
It's also possible to have app_id if you want to do so:
```suggestion
  app_id?: string;
  manifest: string;
```
reference: https://api.slack.com/methods/apps.manifest.validate